.travis.yml

```diff
@@ -38,7 +38,13 @@ jobs:
     # - stage: automerge
     #   if: env(PR_AUTHOR)=openshift-cherrypick-robot
     #   script: bash ./automerge.sh
+    - stage: netlify
+      language: minimal
+      if: branch IN (main, enterprise-4.10, enterprise-4.11)
```

Comment:
@kalexand-rh We need to decide which branches should we enable previews for. 

@pneedle-rh We also need to update Travis to handle 4.12 branch:
- https://github.com/openshift/openshift-docs/blob/main/.travis.yml#L18
- https://github.com/openshift/openshift-docs/blob/main/.travis.yml#L28
to make it `if: branch IN (main, enterprise-4.11, enterprise-4.12)`

Reply:
@gaurav-nelson thank you for highlighting that. I have subsequently created https://github.com/openshift/openshift-docs/pull/49637 to address that requirement.